config/default.json

```diff
@@ -211,6 +211,9 @@
         "AllowCustomThemes": true,
         "AllowedThemes": []
     },
+    "DisplaySettings": {
+        "EnableTimezoneSelection": false
```

Comment:
👍 We've needed a section like this for such a long time. We keep jamming this type of setting into ServiceSettings even though it there's so much random stuff in there now that it makes sense to separate this out